internal/plugins/ansible/v1/scaffolds/internal/templates/requirements.go

```diff
@@ -34,6 +34,6 @@ func (f *RequirementsYml) SetTemplateDefaults() error {
 const requirementsYmlTmpl = `---
 collections:
   - name: community.kubernetes
-    version: "<1.0.0"
+    version: ">=1.1.0"
```

Comment:
This is going to have to be a patch in the patches directory before we can merge this.

Reply:
https://github.com/openshift/ocp-release-operator-sdk/blob/master/README.md#patching